### PR TITLE
Rename accessibility_options_used -> accessibility_options in Trips payload

### PR DESCRIPTION
**Pull request by avatarneil**
Also add more descriptive description :)

### CODE REVIEW

File: agency/README.md [hunk 1]
```diff
@@ -365,20 +365,20 @@ The reservation method enum describes the different ways in which a passenger ca
...
+| cancellation_reason   | string        | Conditionally Required | The reason why a *driver* cancelled a reservation. (required if a driver cancelled a trip, and a `driver_cancellation` event_type was part of the trip)                                                                                                                                                            |
+| fare                  | [Fare](#fare) | Conditionally Required | Fare for the trip (required if trip was completed)                                                                                                                                                                                                                                                                 |
+| accessibility_options | Enum[]        | Optional               | The **union** of any accessibility options requested, and used. E.g. if the passenger requests a vehicle with `wheelchair_boarding`, but doesn’t utilize the features during the trip, the trip payload will include `accessibility_options: ['wheelchair_boarding']`. See [accessibility-options][accessibility-options] |
```
Comment (philipjrose) [inline agency/README.md]: Should this be `wheelchair_accessible` instead of `wheelchair_boarding` as the example value?

Comment (avatarneil) [reply agency/README.md]: Derp, yes. I keep getting confused with which one we're using, heh.

File: agency/README.md [hunk 2]
```diff
@@ -378,7 +378,7 @@ The reservation method enum describes the different ways in which a passenger ca
...
 | fare                  | [Fare](#fare) | Conditionally Required | Fare for the trip (required if trip was completed)                                                                                                                                                                                                                                                                 |
-| accessibility_options | Enum[]        | Optional               | The **union** of any accessibility options requested, and used. E.g. if the passenger requests a vehicle with `wheelchair_boarding`, but doesn’t utilize the features during the trip, the trip payload will include `accessibility_options: ['wheelchair_boarding']`. See [accessibility-options][accessibility-options] |
+| accessibility_options | Enum[]        | Optional               | The **union** of any accessibility options requested, and used. E.g. if the passenger requests a vehicle with `wheelchair_accessible`, but doesn’t utilize the features during the trip, the trip payload will include `accessibility_options: ['wheelchair_accessible']`. See [accessibility-options][accessibility-options] |
```
Comment (brianngca) [inline agency/README.md:381]: What is supposed to be in the payload if the accessibility option is requested AND is used?

The link "accessibility-options" redirects to the agency spec, but that doesn't contain any mention of accessibility-options

Comment (avatarneil) [reply agency/README.md:381]: The link ought to redirect to `general-information.md`, hm. In the case that it's requested *and* used, it's inclusive, so it should just be the same as if it was only requested.

**Pull request merged**